src/lang/isEmpty.js

```diff
@@ -0,0 +1,18 @@
+const isEmpty = (value) => {
+  if(typeof value === 'number'){
+    return true;
+  } else if (typeof value === 'string' || Array.isArray(value)){
+    if(value.length === 0){
+      return true;
+    }
+      return false;
+  } else if (typeof value === 'object'){
+    if(Object.keys(value).length === 0){
+      return true;
+    }
+      return false;
+  }
+}
+
```

Comment:
The logic here can be condensed to not be so verbose.
You're checking if something is an enumerable object that is not empty so I feel like the outermost check should be:
```
// check to see if it is an object or a string
if( typeof value === 'object' ||  typeof value === 'string' ) {
  // do some other things to check if value is empty or not
} else {
  return false
}
```
The reason why you don't need to check for `Array.isArray(value)` is because arrays are objects.